atmo/models.py

```diff
@@ -56,14 +58,26 @@ def remove(self):
 
 class EditedAtModel(models.Model):
 
-    created_at = models.DateTimeField(editable=False, blank=True, auto_now_add=True)
-    modified_at = models.DateTimeField(editable=False, blank=True, auto_now=True)
+    created_at = models.DateTimeField(
+        editable=False,
+        blank=True,
+        default=timezone.now,
+    )
+    modified_at = models.DateTimeField(
+        editable=False,
+        blank=True,
+        default=timezone.now,
+    )
```

Comment:
So basically auto_now & auto_now_add suck since they work on model field level and are only ever applied once the object is stored, during the call of the individual field's `pre_save` method. That leaves the object in a weird state where other fields with a default have already set the instance attribute but the auto_now/auto_now_add fields haven't.

A long time ago (in Django 1.0 times) we tried to remove both from Django but failed due to a release schedule conflict. Over the years too many people have expected this weird behavior and now we're stuck with it. The better alternative is to just use the `default` param and override the `save` method for `auto_now`.

But to answer your question, this was triggered by an [issue in factory_boy](https://github.com/FactoryBoy/factory_boy/issues/102) that would prevent the fake object's attributes to be overridden if needed.